railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -7,11 +7,17 @@ module ActionMethods # :nodoc:
     attr_reader :options
 
     def initialize(generator)
-      @generator = generator
-      @options   = generator.options
+      if generator.options[:master]
+        puts '== Generating new rails app based off of rails/rails master branch =='
+        MasterGemfileGenerator.new
```

Comment:
Hm, not exactly liking this call structure, but I don't know where else to put this.